app/models/miq_event.rb

```diff
@@ -21,10 +21,7 @@ class MiqEvent < EventStream
                                         ContainerNode, ContainerImage, PhysicalServer].freeze
 
   CLASS_GROUP_LEVELS = [MiqPolicy::CONDITION_SUCCESS, MiqPolicy::CONDITION_FAILURE].collect { |level| level.downcase.to_sym }
-
-  def self.description
-    _("Management Events")
-  end
```

Comment:
☝️ Is already defineed on line 31 and the above is wrong

Reply:
Good find.